### ISSUE TITLE
Latest release regressed on matching USB devices

### ISSUE THREAD

**Issue by robarnold** [squat/generic-device-plugin#55]
I think the USB detection may have regressed a bit with the recent changes to the USB serial logic. I am seeing an issue on [the latest version](https://github.com/squat/generic-device-plugin/pkgs/container/generic-device-plugin/144052638?tag=amd64-ce28aa70e236e5ec5a5e58f788778b01facb6b8f) that was not seen on [the one just prior](https://github.com/squat/generic-device-plugin/pkgs/container/generic-device-plugin/142980035?tag=235f1d3b089122a490b86531bc45a54cc59a6b92). I can reliably confirm by pinning the version to one or the other.

Here are my relevant args on the CLI for the `DaemonSet` pods:
```
        - '--device'
        - |
          name: coraltpu
          groups:
            - usb:
                - product: 089a
                  vendor: 1a6e
            - usb:
                - product: 9302
                  vendor: 18d1
```

**Comment by squat** [comment]
Thanks for reporting @robarnold! This will be fixed by #54

**Comment by squat** [comment]
Closed by #54
